apps/opik-frontend/src/types/llm.ts

```diff
@@ -35,7 +36,15 @@ export interface LLMMessage {
   autoImprove?: boolean;
 }
 
-export type ProviderMessageType = Omit<LLMMessage, "id">;
+// Backend API format has separate fields for string vs array content
+export type ProviderMessageType = {
+  role: LLM_MESSAGE_ROLE;
```

Comment:
minor - why not re-use LLMMessage as previous ? and add whats needed . can be improved later